### ISSUE TITLE
[Feature Request] - Keyboard Navigation

### ISSUE THREAD

**Issue by Athanasius** [koreldan/ed-map#1]
Apologies if I just failed to find such .... Could you add some keyboard navigation?

WASD / Cursors - move around the plane of the galaxy.  Use a modifier (one of Shift/Ctrl/Alt) to move faster.  Use another to rotate the view instead.
PgUp/Dn - move up/down.
Home/End - Zoom in/out.
Double some of that up: Q/E rotate around 'pole'.  R/F for up/down. Z/X for zoom in/out.

**Comment by koreldan** [comment]
working on...
